test/username-decorator.test.js

```diff
@@ -0,0 +1,35 @@
+import chai from 'chai';
+import { decorateBareUsernameWithAt } from '../lib/helpers';
+chai.should();
+
+describe('Bare Slack Username Replacement', function() {
```

Comment:
I can change them to arrow functions; I didn't see you using them elsewhere so I stayed with old-style `function` instead.